### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by xialonglee**
`Cahce-Controle` should be `Cahce-Control`

### CODE REVIEW

File: docs/design/i18n.md [hunk 1]
```diff
@@ -35,4 +35,4 @@
 
 ### Serving the localized index.html
 * In production, we use our GOLANG backend as a webserver (serves the pages). In development we do not (we serve via browsersync). We do not have internationalization when serving in development mode.
-* In production, the GO backend's server component intercepts all requests and checks their "Accept-Language" header, to determine the locale of the user. Based on the locale, the server chooses from which of the `./dist/<locale>` directories to serve. To avoid caching issues in browsers, the "index.html" pages are never cached (header `Cahce-Controle: no-cache` in the requests).
+* In production, the GO backend's server component intercepts all requests and checks their "Accept-Language" header, to determine the locale of the user. Based on the locale, the server chooses from which of the `./dist/<locale>` directories to serve. To avoid caching issues in browsers, the "index.html" pages are never cached (header `Cahce-Control: no-cache` in the requests).
```
Comment (rf232) [inline docs/design/i18n.md]: Ooh, thanks, but there is more, it should be Cache-Control, can you also fix that perhaps :)

Comment (xialonglee) [inline docs/design/i18n.md]: @rf232 I have tried to search every text files in this repository, but seems only this item was misspelled.

Comment (rf232) [inline docs/design/i18n.md]: the problem is that it was misspelled in two ways, it is Ca**ch**e not Ca**hc**e. I only saw this when you fixed the problem in the Controle -> Control fix :)

**Comment by xialonglee** [comment]
@rf232 oh... i misunderstood your mean. ok , i fix another misspelled word. XD

**Comment by rf232** [comment]
Thanks for your attention to detail!

**Pull request merged**